<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infoticket</class>
 <widget class="QMainWindow" name="infoticket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>751</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="conteneur">
     <item row="0" column="0">
      <widget class="QLabel" name="sujetLbL">
       <property name="text">
        <string>Sujet du ticket :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="sujetData">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLbl">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="emetteurLbl">
       <property name="text">
        <string>Emetteur</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="emetteurData">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="typeLbl">
       <property name="text">
        <string>Type de ticket</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="typeData">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="urgenceLbl">
       <property name="text">
        <string>Niveau d'urgence</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="urgenceData">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="commentaireLbl">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Commentaire de&lt;/p&gt;&lt;p&gt;l'administrateur&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPlainTextEdit" name="commentaireData"/>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="valider_bout">
       <property name="text">
        <string>Valider le bon traitement de la commande</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="descriptionData">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="clubLbl">
       <property name="text">
        <string>Son Club</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="clubData">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="introduction">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
